### DISCUSSION TITLE
Separate albums and "appears on" on the artist page

### DISCUSSION THREAD

**Discussion by silkyclouds** [navidrome/navidrome#2466] [Ideas]
The following display makes it hard to identify the albums : 

![image](https://github.com/navidrome/navidrome/assets/23377067/31d40e82-73b3-4c3c-a475-d8898f67a279)

Is there by any chance a way to display the albums first, then the EPs, then the singles, and finally the compilations for each artist ?

**Comment by certuna** [comment]
This PR is currently pending: https://github.com/navidrome/navidrome/pull/2413 . It sorts the artist's own releases before his compilation appearances, see screenshot below. This helps a lot to make the Artist page more logical for artists with tons of compilation tracks.

Support for Release Type (album, single, ep, live, dj-mix etc) is planned but displaying them nicely in different categories on the artist page like Plex/Spotify/etc will probably have to wait until after the migration to the new [React Admin 4](https://marmelab.com/blog/2022/04/13/react-admin-v4.html) UI framework, no use putting a lot of work into making something for the old UI.

![Screenshot 2023-08-04 at 15 25 39](https://github.com/navidrome/navidrome/assets/62144283/bb3e7131-470b-4d3c-b49e-3021cc157238)

**Comment by zihan-huang** [comment]
Is there any progress on distinguishing albums ,EPs and the singles?
